src/core_plugins/kibana/public/visualize/wizard/wizard.js

```diff
@@ -47,13 +47,27 @@ module.controller('VisualizeWizardStep1', function ($scope, $route, kbnUrl, time
   kbnUrl.removeParam(DashboardConstants.ADD_VISUALIZATION_TO_DASHBOARD_MODE_PARAM);
 
   const visTypes = Private(VisTypesRegistryProvider);
+  $scope.labModeEnabled = config.get('visualize:labMode');
```

Comment:
When removing the commented code in the template, this doesn't need to be in the `$scope` anymore and can be a local variable.